files/models.py

```
@@ -122,7 +122,16 @@ def topic_thumb_path(instance, filename):
     file_name = "{0}.{1}".format(friendly_token, helpers.get_file_name(filename))
     return settings.MEDIA_UPLOAD_DIR + "topics/{0}".format(file_name)
 
+class Language(models.Model):
+    code = models.CharField(max_length=100, help_text="language code")
+    title = models.CharField(max_length=100, help_text="language code")
+
+    class Meta:
+        ordering = ["title"]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)